### ISSUE TITLE
Sinatra master breaks namespace spec

### ISSUE THREAD

**Issue by zzak** [sinatra/sinatra-contrib#200]
1. `$ sinatra=master bundle`
2. `$ sinatra=master bundle exec rake spec`

Produces the following errors:

```
Failures:

  1) Sinatra::Namespace HTTP GET error handling falls back to the handler defined in the base app
     Failure/Error: last_response.body.should        eq 'nicht gefunden' unless verb == :head

       expected: "nicht gefunden"
            got: "not found..."

       (compared using ==)
     # ./spec/namespace_spec.rb:515:in `block (5 levels) in <top (required)>'

  2) Sinatra::Namespace HTTP POST error handling falls back to the handler defined in the base app
     Failure/Error: last_response.body.should        eq 'nicht gefunden' unless verb == :head

       expected: "nicht gefunden"
            got: "not found..."

       (compared using ==)
     # ./spec/namespace_spec.rb:515:in `block (5 levels) in <top (required)>'

  3) Sinatra::Namespace HTTP PUT error handling falls back to the handler defined in the base app
     Failure/Error: last_response.body.should        eq 'nicht gefunden' unless verb == :head

       expected: "nicht gefunden"
            got: "not found..."

       (compared using ==)
     # ./spec/namespace_spec.rb:515:in `block (5 levels) in <top (required)>'

  4) Sinatra::Namespace HTTP DELETE error handling falls back to the handler defined in the base app
     Failure/Error: last_response.body.should        eq 'nicht gefunden' unless verb == :head

       expected: "nicht gefunden"
            got: "not found..."

       (compared using ==)
     # ./spec/namespace_spec.rb:515:in `block (5 levels) in <top (required)>'

  5) Sinatra::Namespace HTTP OPTIONS error handling falls back to the handler defined in the base app
     Failure/Error: last_response.body.should        eq 'nicht gefunden' unless verb == :head

       expected: "nicht gefunden"
            got: "not found..."

       (compared using ==)
     # ./spec/namespace_spec.rb:515:in `block (5 levels) in <top (required)>'

  6) Sinatra::Namespace HTTP PATCH error handling falls back to the handler defined in the base app
     Failure/Error: last_response.body.should        eq 'nicht gefunden' unless verb == :head

       expected: "nicht gefunden"
            got: "not found..."

       (compared using ==)
     # ./spec/namespace_spec.rb:515:in `block (5 levels) in <top (required)>'

Finished in 4.23 seconds
856 examples, 6 failures
```

[travis log](https://travis-ci.org/sinatra/sinatra-contrib/jobs/127503780#L1012)

**Comment by zzak** [comment]
Essentially all of the namespace tests fail now: https://travis-ci.org/sinatra/sinatra-contrib/jobs/127980688

**Comment by stjhimy** [comment]
It's fixed now, I guess the error on Sinatra side: 

```
~/C/o/sinatra-contrib git:master ❯❯❯ git reset --hard origin/master                                                    
HEAD is now at 4e0f937 Remove duplicate `tag` command from release task
~/C/o/sinatra-contrib git:master ❯❯❯ sinatra=master bundle update                                              
``` 

```
sinatra=master bundle exec rspec
Finished in 4.08 seconds
837 examples, 0 failures
```

**Comment by stjhimy** [comment]
I will work on this next week, it seems still failing with Sinatra master.

**Comment by zzak** [comment]
@stjhimy Any help is appreciated here! This is the major blocker atm

**Comment by zzak** [comment]
Fixed by #207
